src/index.js

```diff
@@ -246,6 +260,34 @@ Adapter.extend({
   },
 
   /**
+   * Convert mongodb _id to id
+   *
+   * @method MongoDBAdapter#_convertMongo_id
+   * @return {*} converts the params _id key(s) to id
+   */
+  _convertMongo_id (d, opts) {
+    if (!d) return
+    opts || (opts = {})
+    if (!this.getOpt('convertMongo_id', opts)) return d
+
+    const convert = (x) => {
```

Comment:
Can we pull this out into a top-level standalone function? It doesn't look like it needs to create a closure over anything.